### PR TITLE
Delay feedback on splash

### PR DESCRIPTION
**Pull request by gabriel**
This seems like a safe way to have the feedback show after 4s delay.

Disccused here: https://keybase.atlassian.net/browse/DESKTOP-3820

### CODE REVIEW

**Comment by gabriel** [comment]
@keybase/react-hackers

File: shared/login/forms/index.native.js [hunk 1]
```diff
@@ -1,27 +1,45 @@
...
-    </Box>
-  </Box>
-)
+class Splash extends Component<void, Props, {showFeedback: boolean}> {
+  state = {
+    showFeedback: false,
+  }
+  componentWillMount() {
+    if (!this.state.showFeedback) {
+      setTimeout(() => {
```
Comment (chrisnojima) [inline shared/login/forms/index.native.js]: you should use the timershoc which has checks for if you're mounted or not, else if you transition away and this thing is gone when settimeout happens it'll crash

Comment (chrisnojima) [inline shared/login/forms/index.native.js]: https://github.com/keybase/client/blob/master/shared/common-adapters/hoc-timers.js

**Comment by gabriel** [comment]
Using HOCTimers, thanks

File: shared/login/forms/index.native.js [hunk 2]
```diff
@@ -1,26 +1,48 @@
...
-      </Text>
-      <Button
-        label="Let us know"
-        onClick={props.onFeedback}
-        style={{marginTop: globalMargins.small}}
-        type="Primary"
-      />
-    </Box>
-  </Box>
+const Splash = HOCTimers(
```
Comment (chrisnojima) [inline shared/login/forms/index.native.js]: just fyi you could use recompose and not have to build the class, you can use compose(withState, lifecycle) etc

**Pull request merged**